PROVESFlightControllerReference/test/int/radio_test.py

```
@@ -53,3 +58,59 @@ def test_01_transmit_enabled(fprime_test_api: IntegrationTestAPI, start_gds):
+        fprime_test_api,
+        f"{lora}.CONTINUOUS_WAVE",
+        [CW_SECONDS],
+    )
+
+    # Wait out the CW duration so the modem fully tears down before the next call.
+    time.sleep(CW_SECONDS + 2)
+
+    # Second continuous-wave burst. This is the call that failed before the fix
+    # (modem left wedged after the first CW), so OK here is the key regression signal.
+    start: TimeType = TimeType().set_datetime(
+        datetime.now(), time_base=TimeType.TimeBase("TB_DONT_CARE")
```

Ruff (0.15.21)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)